gramex/cache.py

```diff
@@ -128,6 +128,45 @@ def _template(path, **kwargs):
     return tornado.template.Loader(root, **kwargs).load(name)
 
 
+def excel_reader(path, sheet_name='Sheet1', range=None, name=None, table=None, header=0):
+    # import openpyxl
+    # Read the file using openpyxl
+    # If range (e.g. A10:C30) is specified, return cells in that range
+
+    # If name is specified, return cells in that named range using wb.defined_names
+    # If table is specified, return cells in that named table using ws.tables
+    # Convert into a DataFrame -- using header if specified. header can be int, list of int, None
```

Comment:
What would happen if you use both `range` and `name` and or `table`? If it's only intended to handle exclusive cases, `if, else if` control logic might be appropriate here no?

Reply:
@pratapvardhan We are yet to decide the behavior in such scenarios. @sanand0 Could you please provide your thoughts?

Reply:
table overrides name overrides range. I've implemented this